packages/addons/service/vdr-addon/source/resources/settings.xml

```diff
@@ -32,6 +32,7 @@
 		<setting id="ENABLE_XMLTV2VDR" type="bool" label="1045" default="false" />
 		<setting id="ENABLE_DUMMYDEVICE" type="bool" label="1047" default="false" />
 		<setting id="ENABLE_SATIP" type="bool" label="1048" default="false" />
+		<setting id="SATIP_NUM_DEVICES" type="slider" option="int" range="1,4" label="1058" visible="!eq(-1,false)" default="1" />
```

Comment:
@frankr612 could you adjust the range to 8 or 10 as at least 8 tuner Sat>IP devices are out there and self made are easily >4. If you are unaware how to squash your commits we can do this at merge time !